MCprep_addon/mcprep_ui.py

```diff
@@ -845,6 +903,10 @@ class MCPREP_PT_bridge(bpy.types.Panel):
 	bl_category = "MCprep"
 
 	def draw(self, context):
+		if addon_just_updated():
```

Comment:
It would be nice if there was a short comment above these explaining what they do for any future developers

Reply:
Do you think it's sufficient to have that comment in the code of those functions? Otherwise we'll be having copies of the comment in 10x places, adding to the boilerplate.

Reply:
Should be fine as long as it's explained what the functions do and why they do that they do

Reply:
So... would you modify the docstrings that are there now I guess is the question :)